packages/angular_devkit/build_angular/src/tslint/index.ts

```diff
@@ -134,6 +139,7 @@ async function _run(
 }
 
 
+/** @deprecated Deprecated as part of the TSLint deprecation. */
```

Comment:
Can you add the version number for the deprecation to the comments as well?

Reply:
Yeah, even in the schema?

Reply:
probably just the code comments.